test/simulation_core.jl

```diff
@@ -0,0 +1,341 @@
+module SimulationCoreTest
+
+using ACSets
+using Decapodes
+using DiagrammaticEquations
+using MLStyle
+using Test
+
+const EQUALS = :(=)
+const DOT_EQUALS =:.=
+
+######################
```

Comment:
When we add linting rules, these section headers will get changed.

Reply:
Not if I add the linted rules :)